drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/lockmanager/GuidedDecisionTableLockManagerImplTest.java

```diff
@@ -137,6 +137,24 @@ public void testFireChangeTitleEvent_LockInfoUpdateForActiveDecisionTable() {
                times(1)).fire(any(ChangeTitleWidgetEvent.class));
     }
 
+    @Test
+    public void testFireChangeTitleEvent_LockInfoUpdateForActiveDecisionTableMissingCurrentPath() {
+        lockManager.init(mock(LockTarget.class),
+                         modellerPresenter);
+
+        final GuidedDecisionTableView.Presenter dtPresenter = mock(GuidedDecisionTableView.Presenter.class);
+        final PathFactory.PathImpl lockInfoPath = makePathImpl("filename");
+
+        when(lockInfo.getFile()).thenReturn(lockInfoPath);
+        when(dtPresenter.getCurrentPath()).thenReturn(null);
```

Comment:
@jomarko Whilst this test does no harm; it should be impossible to have a ```GuidedDecisionTableView.Presenter``` instance without "current path" set. It might be worth checking/adding there's a test for that too?

Reply:
@manstis I am lost, I think this test checks what happens if current path would be null, what test should I look for please?